features/product/viewing_products/viewing_different_discounted_price_for_different_product_variants.feature

```diff
@@ -23,3 +23,11 @@ Feature: Viewing different discounted price for different product variants
         And I select "Wyborowa Apple" variant
         Then the product variant price should be "$12.55"
         And the product original price should be "$20.00"
+
+    @todo
+    Scenario: Viewing a detailed page with product's discounted catalog price for different variant
+        Given there is a catalog promotion "For apple drinks" that reduces price by "30%" and applies on "Wyborowa Apple" variant
+        When I view product "Wyborowa Vodka"
+        And I select "Wyborowa Apple" variant
+        Then the product variant price should be "$8,79"
+        And the product original price should be "$20.00"
```

Comment:
I wonder, should we use `price/original price` words within the shop context. For a Customer/Visitor it's more like a `discounted price/price`, wdyt?

Reply:
IMO the most important thing here is still `Price`, and this is something that really bothers Mr. Customer. The way how we say what was the price before is not so important - does the customer cares if it was called original? I don't think so. And in this case I would think about API first - where we are returning original price 🎉

Reply:
Yes.... but! 😄 API-first is not an argument, as Behat scenarios are and should be always context-agnostic. What we should do in Behat scenarios is imitating the user behaviour. So, does the Customer sees the crossed-out price with a new one beside and thinks "the original price is X, but a price is now Y!"? Or it's more like "the price before was X, but a new, discounted price is Y!" 🚀 🐎 

Of course, it's not a blocker for merge, but I believe we should really focus on Customer's behaviour and their **vocabulary**, to write perfect Behat scenarios :) 🖖